### ISSUE TITLE
Al issues #3, #4, #5

### ISSUE THREAD

**Issue by KSmithO** [Section508Coordinators/DHSA-TT-200-A#117]
From: allen.hoffman@deque.com <allen.hoffman@deque.com> 
Sent: Thursday, June 13, 2019 2:54 PM
To: Accessibility <Accessibility@HQ.DHS.GOV>
Subject: ATT200A pilot feedback Allen Hoffman #3

The title of the incremental practice page says certification.  It should not say certification.

From: allen.hoffman@deque.com <allen.hoffman@deque.com> 
Sent: Thursday, June 13, 2019 2:57 PM
To: Accessibility <Accessibility@HQ.DHS.GOV>
Subject: ATT200A pilot feedback Allen Hoffman #4

In the incremental practice exam I see:

Please review the incorrect answered Test IDs below. The incorrectly answered Test IDs will be emailed to the address on file.
Before you attempt to retake the exam, please select one of the options below:
Option 1:  Instructor Post-OFTE251 Review (Recommended)
To review your test results with an instructor, please complete the following steps.

You need to make this feedback relevant to incremental exams and ofte251 doesn’t exist anymore.

From: allen.hoffman@deque.com <allen.hoffman@deque.com> 
Sent: Thursday, June 13, 2019 3:02 PM
To: Accessibility <Accessibility@HQ.DHS.GOV>
Subject: ATT200A pilot feedback Allen Hoffman #5

In resources I see references to old 508 software standards.
I have not checked the whole list but suspect there are a lot of old references in resources page which need replacing.

**Comment by KSmithO** [comment]
known issues-pending Carney fix

**Comment by mikebarklage** [comment]
Text updated on Dev, waiting for approval.

**Comment by Tonylayton1** [comment]
verified on dev

**Comment by AnnMarieDavis** [comment]
Three issues identified:
1. Incremental practice exams are labeled as "certification exam" in both the blue banner and the grey strip below the banner. This has NOT been fixed. A separate issue has been opened for this:  #176 
2.  Remove incorrect instructions to review the results with an instructor.  I attempted Topic 15 and put in no correct answers.  I did not receive the incorrect instructions on any of my failed attempts. 
3. Resources link has been reviewed from every incremental exam.
